docs/content/guide/component.ngdoc

```diff
@@ -388,7 +388,7 @@ Here is a tab pane example built from components:
 angular.module('docsTabsExample', [])
   .component('myTabs', {
     transclude: true,
-    controller: function() {
+    controller: function Controller() {
```

Comment:
Nitpick: Since this appears in the guide (which is supposed to promote good practices), I would name this `MyTabsController`.